cmd/platform.go

```diff
@@ -31,6 +31,10 @@ func configurePlatform(cmd *cobra.Command) {
 	flags.BoolP("include-subgroups", "", false, "Include GitLab subgroups when using the --group flag.")
 	flags.BoolP("ssh-auth", "", false, `Use SSH cloning URL instead of HTTPS + token. This requires that a setup with ssh keys that have access to all repos and that the server is already in known_hosts.`)
 
+	// This is only used by PrintCmd to mark readOnly mode for version control platform
+	flags.Bool("readOnly", false, "If set, This is running in readonly will be read-only.")
+	_ = flags.MarkHidden("readOnly")
```

Comment:
This is added inside `configurePlatform` which is run for all cmds.
Marking it hidden as this is set programmatically by the print command.